Bangazon/dbUtilities.cs

```diff
@@ -191,6 +226,46 @@ FOREIGN KEY(`productID`) REFERENCES `productOrder`(`id`)
                 }
                 _connection.Close ();
             }
-        }   
+
+        }
+        // Checks to see if PaymentType exists, if it doesn't it creates the table.
```

Comment:
This is also not a big deal, but this code is basically being inserted into two different PRs. Thankfully your code is good, but what if there were bugs in this part? It would have to be fixed in two different PRs and hold back key functionality from being merged. Better to either group them together or else write both branches and wait until one of them has been approved/merged before PRing the second one. Just IMO.